Jenkinsfile

```diff
@@ -96,10 +97,16 @@ stage("Build Docs") {
         } else {
             isMaster = env.BRANCH_NAME == 'master'
             bucket = 'autogluon.mxnet.io'
-            path = isMaster ? '' : "${env.BRANCH_NAME}/"
+            path = isMaster ? 'dev/' : "${env.BRANCH_NAME}/"
```

Comment:
what happens if the branch is 'dev'? Do we just avoid having a branch with that name?

Reply:
If we have dev branch, it will overwrite this space. So the recommendation is to avoid this branch name.

Reply:
for safety, could we have isDev and then set path to 'dev-branch/' just to avoid worst case scenario? It will be hard for all team members to always remember this pitfall.

Reply:
good point, added `dev-branch` convention